docker_files/consume_ztf.Dockerfile

```diff
@@ -1,23 +0,0 @@
-FROM python:3.7
```

Comment:
This file is deleted, but the image is still in the GCP container registry.

Reply:
As far as I'm concerned we can delete that image from the registry, along with all the other images that are in there.
However, they are not interfering with anything.

Reply:
Done.